draft-ietf-httpbis-http2bis.xml

```diff
@@ -3086,15 +3091,15 @@
             frames.
           </t>
           <t>
-            Trailing header fields are sent as a header block after both the request or response
-            header block and all the <xref target="DATA" format="none">DATA</xref> frames have been sent.  The
-            <xref target="HEADERS" format="none">HEADERS</xref> frame starting the trailers header block has the END_STREAM flag
-            set.
+            A trailer section is sent as a field block after both the request or response
+            field block and all the <xref target="DATA" format="none">DATA</xref> frames have been sent.  The
+            <xref target="HEADERS" format="none">HEADERS</xref> frame starting the field block that comprises
+            the trailer section has the END_STREAM flag set.
```

Comment:
Is this a MUST and/or is the message malformed otherwise?

Reply:
There are MUSTs in other places.